modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -124,11 +124,19 @@ Double the space if the server is an ISS Master.
 
 |===
 
+endif::[]
 
 
-endif::[]
+[IMPORTANT]
+====
+{productname} performance depends on a number of factors, for example, underlying resources, network bandwidth, latency between clients and server etc.
 
+Based on the experience and different deployments that are in use, the advice for optimal performance of {productname} Server with an adequate number of proxies is to not exceed 10.000 clients per single server.
+It is highly recommended to move to the Hub setup and involve consultancy when you have more than 10.000 clients.
+Without fine tuning and an adequate number of proxies, such a large number of clients can lead to performance issues.
```

Comment:
I would put something like this since we are talking about numbers above 10.000. What do you think @admd 
```suggestion
Even with fine-tuning and an adequate number of proxies, such a large number of clients can lead to performance issues.
```